.github/workflows/cron.yml

```diff
@@ -119,7 +119,7 @@ jobs:
     if: github.repository == 'Project-MONAI/MONAI'
     strategy:
       matrix:
-        container: ["pytorch:21.02", "pytorch:21.10", "pytorch:22.12"]  # 21.02, 21.10 for backward comp.
+        container: ["pytorch:22.12"]
```

Comment:
May I know why remove them?

Thanks.

Reply:
Those are deprecating tests for early versions of clara train 4.0/4.1, I don't think we should still spend resources testing...

Reply:
Please also add 22.11 here, this is will be part of the nightly tests. @YanxuanLiu

Reply:
OK, I see.

Thanks.